tests/virt_autotest/login_console.pm

```diff
@@ -70,8 +70,26 @@ sub login_to_console {
     save_screenshot;
     send_key 'ret';
 
-    assert_screen(['linux-login', 'virttest-displaymanager'], $timeout);
+    my $host_installed_version = get_var('VERSION_TO_INSTALL', get_var('VERSION', ''));
+    ($host_installed_version) = $host_installed_version =~ /^(\d+)/;
+    if (check_var('offline_upgrade', 'yes') && ($host_installed_version eq '11')) {
+        save_screenshot;
+        assert_screen('sshd-server-started-config', 180);
```

Comment:
I guess we can skip at least 2 save_screenshot calls before and after assert_screen, as it will create those. But that's minor thing, so if you don't want to change it, let's simply merge.

Reply:
Already removed those save_screenshots, including the ones around the newly added assert_screen.